client/src/components/Login.js

```diff
@@ -57,18 +57,57 @@ signupLink:{
 }));
 function Login (){
     const classes = useStyles();
-    const [userEmail,setEmail]=useState('')
-    const [userPass, setPass]=useState('')
+    const [email,setEmail]=useState('');
+    const [password, setPassword]=useState('');
+    const [errors, setErrors]=useState('');
+
+    const validations = () => {
+        let temp = {};
+        temp.email = email ? "" : "Please enter an email.";
+        temp.password = password ? "" : "Please enter a password.";
+        setErrors({ ...temp });
+
+        return Object.values(temp).every(x => x == "");
+    }
+
+    const handleClick = (e) => {
+        e.preventDefault();
+        if (validations()) {
+            fetch("/auth/login", {
+                method: 'POST',
+                headers: {
+                    'Content-Type': 'application/json'
+                },
+                body: JSON.stringify({
+                    'email': email,
+                    'password': password,
+                })})
+                .then(response => response.json())
+                .then(function(response) {
+                    if (response.status === 'success') {
+                        console.log('Success:', email);
+                    }
+                    else {
+                        window.alert(response.message);
+                        console.log(response.message);
+                    }
+                })
+                .catch((error) => {
+                    console.error('Error:', error)
+                })
+            }
+    }
+    // The console throws a warning because the 'error' field in each TextField is given a string, not a boolean. However, the code appears to work as intended, and it does not appear to be a problem.
```

Comment:
To fix this, you can coerce the string to a boolean: `error={!!errors.password}`